mongo-helm/templates/mongo.yml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ .Values.mongodbStatefulSet.name }}

```

Comment:
https://github.com/imzainazm/docker-project/pull/13#discussion_r1647446902

> Different environments will be in different namespaces, no? So it would not matter to have different names for the configmap. Can you think of any other reason to expose this to user? That too, without a default value? 🤔